bin/gollum

```diff
@@ -61,6 +61,9 @@ opts = OptionParser.new do |opts|
   opts.on('-r', '--ref [REF]', 'Specify the branch to serve. Default: \'master\'.') do |ref|
     wiki_options[:ref] = ref
   end
+  opts.on("--bare", "Declare '<git-repo>' to be bare. This is only necessary when using the grit adapter.") do
```

Comment:
@dometto Does #1511 show that `This is only necessary when using the grit adapter.` is false?

Reply:
It does. Dang, went too fast -- that notice needs to be removed.

Reply:
I've now fixed this on my local 5.x branch that will be merged into `master`.

Reply:
Excellent.